ControlExtention/src/test/java/RadioButtonTests.java

```diff
@@ -0,0 +1,46 @@
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class RadioButtonTests extends TestBase {
+	private RadioButton yesButton;
+	private RadioButton impressiveButton;
+	private RadioButton noButton;
+	private RadioGroup radioGroup;
+	
+	@BeforeMethod
+	public void launch() {
+		this.driver.navigate().to(baseURL+"radio-button/");
+		radioGroup = new RadioGroup(driver,baseURL+"radio-button/");
+		yesButton = radioGroup.getButton("Yes");
```

Comment:
You may encounter Stale Element Exceptions by doing this.
Don't get the buttons until you are ready to perform some operation on them.
Also don't put code in Constructors or @BeforeMethod that is likely to throw an Exception.